site/zenodo_rdm/openaire/serializers/schema.py

```diff
@@ -156,7 +156,7 @@ def get_license_code(self, obj):
         public_record = access["record"] == "public"
         public_files = access["files"] == "public"
 
-        access_settings = obj.get("parent").get("access").get("settings")
+        access_settings = obj.get("parent").get("access", {}).get("settings", {})
```

Comment:
```suggestion
        access_settings = obj.get("parent", {}).get("access", {}).get("settings", {})
```

Just for those older records we might have around